koku/masu/external/downloader/azure/azure_service.py

```diff
@@ -120,10 +127,7 @@ def _get_latest_blob_for_path(
 
         latest_report = self._get_latest_blob(report_path, blobs, extension)
         if not latest_report:
```

Comment:
Optional side note: Should we change these `latest_report` vars to `latest_file` or similar throughout this function now? Since its not just csv reports that its collecting (Manifests and gzip files)